src/openforms/js/components/admin/form_design/Warnings/MissingAuthCosignWarning.js

```diff
@@ -1,39 +1,27 @@
-import PropTypes from 'prop-types';
 import React, {useContext} from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import MessageList from 'components/admin/MessageList';
 import {FormContext} from 'components/admin/form_design/Context';
 
-const MissingAuthCosignWarning = ({cosignComponent}) => {
-  const {selectedAuthPlugins, plugins} = useContext(FormContext);
+const MissingAuthCosignWarning = () => {
+  const {selectedAuthPlugins} = useContext(FormContext);
 
-  if (selectedAuthPlugins.includes(cosignComponent.authPlugin)) return null;
-
-  const getAuthPluginLabel = pluginId => {
-    for (const plugin of plugins.availableAuthPlugins) {
-      if (plugin.id === pluginId) return plugin.label;
-    }
-
-    return pluginId;
-  };
+  if (selectedAuthPlugins.length > 0) return null;
```

Comment:
how does this affect the cosign v1 component?

Reply:
Urgh, no warning anymore :S I shall add this back

Reply:
Oh! But checking the code, this never showed a warning for the old cosign. I tried it on the test env now, and no warning is displayed for the cosign V1 if the auth plugin is not configured :thinking: